### PR TITLE
Changes to rename Basic to Local.

### PR DESCRIPTION
**Pull request by raraveen**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs [hunk 1]
```diff
@@ -97,5 +97,13 @@ public void TestExpressRouteCircuitPeeringWithRouteFilter()
         {
             TestRunner.RunTestScript("Test-ExpressRouteCircuitPeeringWithRouteFilter");
         }
-    }
+
+	    [Fact(Skip = "Rerecord tests")]
```
Comment (MiYanni) [inline src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs]: So, you added a new test that is... skipped? Why not record it?

File: src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs [hunk 2]
```diff
@@ -59,6 +59,7 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
             MNM.ExpressRouteCircuitSkuTier.Standard,
             MNM.ExpressRouteCircuitSkuTier.Premium,
             MNM.ExpressRouteCircuitSkuTier.Basic,
+            MNM.ExpressRouteCircuitSkuTier.Local,
```
Comment (MiYanni) [inline src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs]: I forgot the history here... is Local replacing Basic? If so, what happens when the user provides Basic?

File: src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs (hunk 1 above)
Comment (raraveen) [reply src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs]: I hit a few issues with the recording. I am working on it. Additionally, the feature is not GA-ed. Spoke with @MikhailTryakhov earlier on this.

File: src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs (hunk 2 above)
Comment (raraveen) [reply src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs]: No worries, we will support both for now will Local being the new name.

File: src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs (hunk 1 above)
Comment (MikhailTryakhov) [reply src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs]: Nono, we need to enable it on test subscription and add the recording.

Overwise - you can request private preview build from powershell

Comment (raraveen) [reply src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs]: Thanks for clarifying. I will fix up the recording. Please go ahead with the branch cut; no need to wait for my merge.

Comment (MikhailTryakhov) [reply src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs]: Thanks for flexibility @raraven. You can retarget to master and be a part of build release or wait for a next monthly branch. Currently closing this pr

**Pull request closed without merging**